aspnetcore/fundamentals/index.md

```diff
@@ -50,87 +50,163 @@ The `UseStartup` method on `WebHostBuilder` specifies the `Startup` class for yo
+A service is a component that's intended for common consumption in an app. Services are made available through [dependency injection (DI)](xref:fundamentals/dependency-injection). ASP.NET Core includes a native **I**nversion **o**f **C**ontrol (IoC) container that supports [constructor injection](xref:mvc/controllers/dependency-injection#constructor-injection) by default. You can replace the default native container if you wish. In addition to its loose coupling benefit, DI makes services available throughout your app (for example, [logging](xref:fundamentals/logging)).
 
 For more information, see [Dependency injection](xref:fundamentals/dependency-injection).
 
 ## Middleware
 
-In ASP.NET Core, you compose your request pipeline using [Middleware](xref:fundamentals/middleware). ASP.NET Core middleware performs asynchronous logic on an `HttpContext` and then either invokes the next middleware in the sequence or terminates the request directly. A middleware component called "XYZ" is added by invoking a `UseXYZ` extension method in the `Configure` method.
+In ASP.NET Core, you compose your request pipeline using [middleware](xref:fundamentals/middleware). ASP.NET Core middleware performs asynchronous logic on an `HttpContext` and then either invokes the next middleware in the sequence or terminates the request directly. A middleware component called "XYZ" is added by invoking an `UseXYZ` extension method in the `Configure` method.
 
 ASP.NET Core comes with a rich set of built-in middleware:
 
 * [Static files](xref:fundamentals/static-files)
-
 * [Routing](xref:fundamentals/routing)
-
 * [Authentication](xref:security/authentication/index)
+* [Response Compression Middleware](xref:performance/response-compression)
+* [URL Rewriting Middleware](xref:fundamentals/url-rewriting)
 
-You can use any [OWIN](http://owin.org)-based middleware with ASP.NET Core, and you can write your own custom middleware.
+[OWIN](http://owin.org)-based middleware is available for ASP.NET Core apps, and you can write your own custom middleware.
 
 For more information, see [Middleware](xref:fundamentals/middleware) and [Open Web Interface for .NET (OWIN)](xref:fundamentals/owin).
 
+## Environments
+
+Environments, such as "Development" and "Production", are a first-class notion in ASP.NET Core and can be set using environment variables.
+
+For more information, see [Working with Multiple Environments](xref:fundamentals/environments).
+
+## Configuration
+
+ASP.NET Core uses a configuration model based on name-value pairs. The configuration model isn't based on `System.Configuration` or *web.config*. Configuration obtains settings from an ordered set of configuration providers. The built-in configuration providers support a variety of file formats (XML, JSON, INI) and environment variables to enable environment-based configuration. You can also write your own custom configuration providers.
```

Comment:
I removed the banded word ~simple~

Reply:
I like that word tho ... in life I mean. :grinning: